<launch>
	<node pkg="tf" type="static_transform_publisher" 
		name="left_cam_tf" args="0 0 1.0 -1.570796 0 -1.570796 /map /left_camera 30"/>
	<node pkg="tf" type="static_transform_publisher" 
		name="right_cam_tf" args="0.0 -0.25 1.0 -1.570796 0 -1.570796 /map /right_camera 30"/>
		
	<!-- Left Camera -->
	<!-- These cameras have 15 fps it seems despite frame rate option-->
	<node name="left_cam" pkg="usb_cam" type="usb_cam_node">
		<param name="video_device" value="/dev/video0" />
		<param name="camera_frame_id" value="left_camera" />
		<param name="pixel_format" value="yuyv" />
		<param name="framerate" value="30" />
		<param name="camera_info_url" type="string" value="file://$(find stereo_color_tracker)/camera_info/left_camera.yaml" />
		<param name="image_width" value="640" />
		<param name="image_height" value="480" />
	</node>

	<!-- Right Camera -->
	<node name="right_cam" pkg="usb_cam" type="usb_cam_node">
		<param name="video_device" value="/dev/video1" />
		<param name="camera_frame_id" value="right_camera" />
		<param name="pixel_format" value="yuyv" />
		<param name="framerate" value="30" />
		<param name="camera_info_url" type="string" value="file://$(find stereo_color_tracker)/camera_info/right_camera.yaml" />
		<param name="image_width" value="640" />
		<param name="image_height" value="480" />
	</node>
<!--	
	<node name="find_target_in_images_node" pkg="stereo_vision" type="find_target_in_images">
	</node>
-->
</launch>
